core/services/run_manager_test.go

```diff
@@ -607,6 +607,7 @@ func TestRunManager_Create_fromRunLogPayments(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			config, configCleanup := cltest.NewConfig(t)
 			defer configCleanup()
+			config.Set("DATABASE_TIMEOUT", "10s") // Lots of parallelized tests
```

Comment:
Feels like a bandaid.

Reply:
Agreed, but I'm not sure what else to do about this except un-parallelize the tests, which would slow CI down massively.